server/utils/file/fileZip.js

```diff
@@ -48,7 +48,10 @@ class FileZip {
 
   async getEntryStream (entryName) {
     return new Promise(resolve =>
-      this.streamZip.stream(entryName, (err, stm) => resolve(stm))
+      this.streamZip.stream(entryName, (err, stm) =>
+        err
+          ? resolve(null)
+          : resolve(stm))
```

Comment:
should it be reject if there's err ?

Reply:
you are right, we should check first if the entry exists, if it doesn't exist we can resolve to null, otherwise reject in case of errors